ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -70,7 +70,18 @@ bgp_ecmp: < number_of_ecmp_paths | default -> 4 >
 # Set to a higher value to allow for very large and complex topologies.
 evpn_ebgp_multihop: < ebgp_multihop | default -> 3 >
 
+
+# EVPN GW ebgp-multihop | Optional
+# Default of 15, considering a large value to avoid BGP reachability issues in very complex DCI networks.
+# Adapt the value for your specific topology.
+evpn_ebgp_gateway_multihop: < ebgp_multihop | default -> 15 >
+
+# EVPN GW inter-domain | Optional
+# Default of True. Enables crossing of BGP EVPN Route Type 5 between different EVPN domains changing the next hop.
+evpn_ebgp_gateway_inter_domain: < true | false | default -> True >
```

Comment:
Q, why do we need a global knob for this one? I suspect it is because you want the default value to be true. I would implement the default `true` in the switch facts instead, and skip this global knob completely.

Reply:
For some reason I though I must explain it somewhere. Changing it